### DISCUSSION TITLE
Sign into app using .local address

### DISCUSSION THREAD

**Discussion by script-js** [immich-app/immich#4042] [Q&A]
I have a late 2014 Mac Mini that I'm using as a server for Immich and BlueBubbles. When I go to the URL I have configured for Immich (immich-server.local) on my PC, it works, but when I try to view it or sign in to the app on my android phone, it can't reach it.

**Comment by bo0tzz** [comment]
Can you reach it from the browser on your phone? Are you using http or https?

**Reply by script-js** [reply]
I'm using http, and I can't access it from my phone browser. It might just be a mac issue.

**Reply by script-js** [reply]
Wait nevermind, I'm going to use a static IP address instead
